### ISSUE TITLE
Returning a Uint8Array doesn't handle the byteOffset

### ISSUE THREAD

**Issue by husseinhere** [plv8/plv8js#540]
```
create or replace function test4() returns bytea as $$
	let ab = new ArrayBuffer(2);
	let view = new Uint8Array(ab);
	view[0] = 6;
	view[1] = 9;
	
	let r = new Uint8Array(ab, 1);
	plv8.elog(NOTICE, 'r is ', r);
	return r; 

$$ language plv8;
```

And calling it:

```
SELECT encode(test4(), 'hex');
```

It correctly logs `r is 9`, but when it's converted into a `bytea` it gets `6`

--- 

I'm guessing here 
https://github.com/plv8/plv8/blob/308c3e499a1072c62708d237897650641ba2f80a/plv8_type.cc#L713-L720
you forget to add byteOffset

**Comment by JerrySievert** [comment]
thank you, this helps.

**Comment by JerrySievert** [comment]
fixed, tests, released 3.1.7.  let me know if you continue to have issues.
